config/base/tests/simple.rs

```diff
@@ -6,7 +6,7 @@ use iroha_config_base::{
 };
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Deserialize, Serialize, Documented, LoadFromEnv)]
+#[derive(Clone, Debug, Deserialize, Serialize, Documented, LoadFromEnv, PartialEq, Eq)]
```

Comment:
In general, if some trait can be derived, it should be derived in library crates. Not so in binary crates.